resources/wikia/libraries/jquery/loadLibrary/jquery.wikia.loadExternalLibrary.js

```diff
@@ -37,7 +26,34 @@
 		}
 
 		return $deferred;
-	};
+	}
+
+	/**
+	 * Load a library via script tag injection.
+	 * Originally adopted from facebook's developer pages but modified for reuse
+	 * Note that it relies on an arbitrary script tag already being in the DOM (which is always the case in our stack)
```

Comment:
Hey, is there a reason why it needs the already existing script tag to work? Maybe the new script could just be inserted into the document.head of without further querying?